apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -194,10 +194,12 @@ defmodule SiteWeb.StopController do
 
   @spec schedules_for_route(Route.t(), Stop.id_t()) :: route_with_directions | nil
   defp schedules_for_route(%Route{} = route, stop_id) do
+    now = Util.now()
```

Comment:
should this also be an opt... or get passed in from a higher level? (for testing purposes)

Reply:
actually, since this is a controller, is now already set in the `conn` somewhere?